### DISCUSSION TITLE
why some files are formated in "symlink",and where is packed_fs.c file，cannot be found

### DISCUSSION THREAD

**Discussion by Mexico-zyy-lost** [cesanta/mongoose#2161] [Ask questions, get answers] [answered]
where to find packed_fs.c，and why there are "symlink" file ?

**Comment by scaprile** [comment, marked as answer]
Please read the docs, and install your tools as instructed.
https://mongoose.ws/documentation/#how-to-build-and-run-examples
https://mongoose.ws/documentation/tutorials/tools/

> _**If you're on Windows 10/11, make sure to enable "Developer mode"**_

**Reply by Mexico-zyy-lost** [reply]
help me a lot！
